config/prow/config.yaml

```diff
@@ -1004,13 +1004,15 @@ presets:
     mountPath: /docker-graph
   - name: docker-root
     mountPath: /var/lib/docker
+  # we need to modprobe ipv6
+  # TODO: move this out of being done in the jobs
+  - mountPath: /lib/modules
+    name: modules
+    readOnly: true
 # volume mounts for kind
 - labels:
     preset-kind-volume-mounts: "true"
```

Comment:
this AND the preset-dind-enabled are currently applied to all kind jobs

so we can move the mount over to the dind mounts, and ensure all dind jobs can modprobe